test/e2e/common/setup-kubestellar.sh

```diff
@@ -138,7 +140,7 @@ kubectl --context imbs1 label managedcluster cluster2 location-group=edge name=c
 : Expect to see the wds1 kubestellar-controller-manager created in the wds1-system 
 : namespace and the imbs1 statefulset created in the imbs1-system namespace.
```

Comment:
This list should be expanded to include the transport controller.

Reply:
fixed